src/extensions/resources/scripts/config.ts

```diff
@@ -49,13 +51,18 @@ export const config: AppConfigInterface = {
 	}),
 	legalLinks: [
 		{
-			url: legalLinks.imprint,
+			url: uiUrl + routePathNames.imprint,
 			label: 'login.legal.infoText.impressum'
 		},
 		{
-			url: legalLinks.privacy,
+			url: uiUrl + routePathNames.privacy,
 			label: 'login.legal.infoText.dataprotection',
 			registration: true
+		},
+		{
+			url: routePathNames.termsAndConditions,
```

Comment:
Die Legal Links werden an mehere Stellen gerendert. Da es vorher keine termsAndConditions gab ist zu klären ob die das brauchen/wollen. Im Profil im Footer usw werden die Links sonst dafür gerendert.

Reply:
Und auch in der Registrierung stand ursprünglich nur "Ich akzeptiere die Datenschutzerklärung ..." da steht jetzt aber "Ich akzeptiere die Datenschutzerklärung und Nutzungsbedingungen ..."

Reply:
Hab die Terms and Conditions rausgenommen, die werden tatsächlich nicht benötigt. Nachdem ich den Nav-Punkt rausgenommen hatte, steht jetzt nur "Ich habe die [Datenschutzerklärung](https://dev.diakonie_frontend.vi.localhost/datenschutz) zur Kenntnis genommen. Für Authentifizierung und Navigation verwendet diese Webseite Cookies."